python/tidy/tidy.py

```diff
@@ -1006,8 +1043,11 @@ def scan(only_changed_files=False, progress=False, github_annotations=False):
     cargo_lock_errors = run_cargo_deny_lints()
     wpt_errors = run_wpt_lints(only_changed_files)
 
+    python_type_check = run_python_type_checker()
```

Comment:
Nit: suggest grouping this with the others:

```suggestion
    python_type_check = run_python_type_checker()
    
```